### ISSUE TITLE
[Feature Request]: Provide a list of images with missing alt text for the lessons team so that those images can be fixed.

### ISSUE THREAD

**Issue by seanlip** [oppia/oppia-android#5044]
**Is your feature request related to a problem? Please describe.**

We would like to figure out which images in the Oppia content have missing alt text so that we can add the alt text on the Web side.

**Describe the solution you'd like**

It should be possible to generate a list of such images.

**Describe alternatives you've considered**
N/A

**Additional context**
This is a dev-facing issue.

**Comment by aadityaguptaa** [comment]
Hi, I'm interested in working on this issue. Before I start, I wanted to clarify: Does the task involve only creating a list of missing alt text descriptions, or will I also be responsible for adding the alt text within the code? Thank you!

**Comment by adhiamboperes** [comment]
This issue only requires listing images missing alt-text, and is best for contributors who have access to the prod assets.
